fe/src/main/java/org/apache/doris/common/util/DynamicPartitionUtil.java

```diff
@@ -148,11 +151,25 @@ private static void checkStartDayOfWeek(String val) throws DdlException {
         }
     }
 
+    private static void checkTimeZone(String val) throws DdlException {
```

Comment:
Why not using `TimeUtils.checkTimeZoneValidAndStandardize()`?

Reply:
In fact, i don't konw the function...